packages/expo-web-browser/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "expo-web-browser",
-  "version": "9.2.0",
+  "version": "9.2.1",
```

Comment:
Please revert this line, update the PR branch and address the Bartek request, so we can merge this bump in. 😉